pkg/operator2/operator.go

```diff
@@ -525,7 +532,8 @@ func (c *authOperator) checkDeploymentReady(deployment *appsv1.Deployment, opera
 func (c *authOperator) checkRouteHealthy(route *routev1.Route, routerSecret *corev1.Secret, ingress *configv1.Ingress) (ready bool, msg, reason string, err error) {
 	caData := routerSecretToCA(route, routerSecret, ingress)
 
-	rt, err := transportFor("", caData, nil, nil)
+	// merge trustedCA data with router cert in case TLS intercept proxy is in place
+	rt, err := transportFor("", append(caData, trustedCABytes()...), nil, nil)
```

Comment:
this reads the file on every request, not good.

Reply:
Are we sure these bytes always append cleanly, i.e. that we have a trailing newline in caData? Looks fragile.

Reply:
ic, we probably reread to get the latest version (which can change in the background).